### ISSUE TITLE
Putting SourceCode to CodeSandbox

### ISSUE THREAD

**Issue by nate-vukovich** [DevExpress/devextreme-reactive#2143]
### I'm using ...
- [x] React Grid

### Description

I wanted to request if it's possible that you put the source code for demos/examples to (https://codesandbox.io/)[https://codesandbox.io/] ?

It would really help to all users and devs to change / see the effects on the fly more easily.

Thanks for all your work and support.

Nejc

**Comment by MaximKudriavtsev** [comment]
Hi @nvwebd,

Thank you for your suggestion, we are planning to implement this feature. We are currently working on a new website and will implement the `open in Codesandbox` button in our demos.

Do you have any suggestion about our website and documentation? Your feedback will be very helpful. We can make the product better together!

**Comment by idrm** [comment]
I agree that accompanying each demo/example with a link to a working setup in CodeSandbox / Codepen would help tremendously.

E.g., in the source code of the [tree grid demo](https://devexpress.github.io/devextreme-reactive/react/grid/demos/featured/tree-data/) there is a reference to a `data.json` file. I had to dig it up by googling the import line to find that the codebase lives [here](https://github.com/DevExpress/devextreme-reactive/tree/master/packages/dx-react-grid-demos/src/demo-sources/grid-featured-tree-data).

**Comment by churkin** [comment]
Hi @idrm,
Thank you for the feedback. We will take your remark into account when updating the site.
